sites/profoodworld/server/routes/website-section.js

```diff
@@ -1,9 +1,14 @@
 const { withWebsiteSection } = require('@base-cms/marko-web/middleware');
 const section = require('../templates/website-section');
 const contactUs = require('../templates/website-section/contact-us');
+const global250 = require('../templates/website-section/global250');
+const global50 = require('../templates/website-section/global-50');
 const queryFragment = require('../graphql/fragments/website-section-page');
 
 module.exports = (app) => {
+  app.get('/global250', (_, res) => { res.marko(global250); });
```

Comment:
Why is one route dasherized and the other not? Just want to make sure this is intentional.